docs/source/extend_kedro/hooks.md

```diff
@@ -13,7 +13,7 @@ Hooks are a mechanism to add extra behaviour to Kedro's main execution in an eas
 A Hook is comprised of a Hook specification and Hook implementation. To add Hooks to your project you will need to:
 
 * Provide a Hook implementation for an existing Kedro-defined Hook specification
-* Register your Hook implementation in the `src/<your_project>/settings.py` file under the `HOOKS` key
+* Register your Hook implementation in the [`src/<your_project>/settings.py`](../kedro_project_setup/settings.md) file under the `HOOKS` key
 
 ### Hook specification
 
```

Comment:
I can't comment on the line on github. But line 20 can be removed, as can the `#### Execution timeline Hooks` section heading now that `#### Registration Hooks` doesn't exist any more.

Reply:
Might be worth just doing a quick docs search for "regist" or something just in case there's any other mention of them anywhere 🙂